lib/usd/translators/mayaReferenceReader.cpp

```diff
@@ -52,8 +52,19 @@ PXRUSDMAYA_DEFINE_READER(MayaUsd_SchemasMayaReference, args, context)
 
     MObject referenceParentNode;
 
-    UsdMayaTranslatorUtil::CreateTransformNode(
-        usdPrim, parentNode, args, &context, &status, &referenceParentNode);
+    // The stand-in for a Maya reference prim is a dummy transform node, which
+    // preserves the source USD prim type in attribute "USD_typeName".  To
```

Comment:
I suppose a future extension would be to use this attribute for any node types that wish to clearly specify the USD node it wishes to come back as?

Reply:
You bet.  I had to adjust the testCustomRig.py test to do this.